client/src/featureform/format.py

```diff
@@ -16,4 +16,23 @@ def format_rows(format_obj, format_obj_2=None, format_obj_3=None, format_obj_4=N
 
 def format_pg(s=""):
     print(divider)
-    print(s)
\ No newline at end of file
+    print(s)
+
+def format_rows_string(format_obj, format_obj_2=None, format_obj_3=None, format_obj_4=None):
```

Comment:
Whats this?

Reply:
Is it possible to make format_rows use this? Take the return from this and print it? Seems like a lot of reused code